standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/ReplChangeManager.java

```diff
@@ -467,4 +470,27 @@ static void scheduleCMClearer(Configuration conf) {
           0, MetastoreConf.getTimeVar(conf, ConfVars.REPLCMINTERVAL, TimeUnit.SECONDS), TimeUnit.SECONDS);
     }
   }
+
+  public static boolean isSourceOfReplication(Database db) {
+    // Can not judge, so assuming replication is not enabled.
+    if (db == null) {
+      LOG.warn("db object passed is null. isSourceOfReplication is returning false");
+      return false;
+    }
+    String replPolicyIds = getReplPolicyIdString(db);
+    return replPolicyIds != null && !replPolicyIds.isEmpty();
```

Comment:
Shall use !StringUtils.isBlank() which checks for both null and empty.

Reply:
done